packages/commons/src/events/ActionDocument.ts

```diff
@@ -158,6 +158,14 @@ export type RequestedCorrectionAction = z.infer<
   typeof RequestedCorrectionAction
 >
 
+const CorrectedAction = ActionBase.merge(
+  z.object({
+    type: z.literal(ActionType.CORRECT)
```

Comment:
I would like to find a way of doing this without needing to configure the same form twice, as it should be the same form in all cases (I think? We could confirm from Jon?). Can we just use the `ActionType.REQUEST_CORRECTION` form, and get rid of this action config?